functions/auth.js

```diff
@@ -0,0 +1,98 @@
+const basicAuth = require('basic-auth');
+const crypto = require('crypto');
+const functions = require('firebase-functions');
+const {
+  fail,
+  STATUS_CODE_UNAUTHORIZED,
+  firestore,
+  isUserType,
+  USER_TYPE_ADMIN,
+  INITIAL_USER_PASSWORD,
+} = require('./common');
+
+const { logger } = functions;
```

Comment:
You can just `const { logger } = require('firebase-functions');` here

Reply:
https://github.com/ably/asset-tracking-backend-demo/pull/11/commits/64af06813916aa4a5d6fad62c52aad3874835fa7